modules/new_serverpod_auth/serverpod_auth_core/serverpod_auth_core_client/lib/src/protocol/protocol.dart

```diff
@@ -16,27 +16,27 @@ import 'auth_user/models/auth_user_model.dart' as _i3;
 import 'auth_user/models/auth_user_not_found_exception.dart' as _i4;
 import 'common/models/auth_strategy.dart' as _i5;
 import 'common/models/auth_success.dart' as _i6;
-import 'jwt/models/authentication_token_info.dart' as _i7;
+import 'jwt/models/jwt_token_info.dart' as _i7;
 import 'jwt/models/refresh_token_expired_exception.dart' as _i8;
 import 'jwt/models/refresh_token_invalid_secret_exception.dart' as _i9;
 import 'jwt/models/refresh_token_malformed_exception.dart' as _i10;
 import 'jwt/models/refresh_token_not_found_exception.dart' as _i11;
 import 'jwt/models/token_pair.dart' as _i12;
 import 'profile/models/user_profile_model.dart' as _i13;
-import 'session/models/auth_session_info.dart' as _i14;
+import 'session/models/server_side_session_info.dart' as _i14;
```

Comment:
Discussion: Should the top level folder still call `session` or also be changed to `sss` to follow the same `jwt` pattern?

```suggestion
import 'sss/models/server_side_session_info.dart' as _i14;
```
> This suggestion is only here to highlight the proposal, not to be applied.